tree/dataframe/src/RDFActionHelpers.cxx

```diff
@@ -204,10 +206,11 @@ void StdDevHelper::Finalize()
       if (fCounts[i] == 0) {
          continue;
       }
-      variance += fCounts[i] * (fDistancesfromMean[i] / fCounts[i] + std::pow((fMeans[i] - overallMean), 2));
+      auto setVariance = fDistancesfromMean[i] / (fCounts[i] - 1);
+      variance += (fCounts[i] - 1) * (setVariance + std::pow((fMeans[i] - overallMean), 2));
    }
 
-   variance = variance / totalElements;
+   variance = variance / (totalElements - totalSets);
```

Comment:
Is it correct? As it is Welford's algorithm (did not fully understand this in my first comment), I'm not sure whether it's biased or unbiased by default (or whether `N -> N - 1` does the trick).

Reply:
@stwunsch Here there `totalSets` non-empty set, each one with a unbiased variance, so the unpooled one can be calculated.